### ISSUE TITLE
Running falcon_unzip error: "ImportError: No module named falcon_kit"

### ISSUE THREAD

**Issue by ls2017** [PacificBiosciences/FALCON_unzip#49]
I got the following error when I ran falcon_unzip:
```py
Traceback (most recent call last):
  File "/usr/local/bin/fc_unzip.py", line 4, in <module>
    __import__('pkg_resources').run_script('falcon-unzip==0.1.0', 'fc_unzip.py')
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 746, in run_script
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 1501, in run_script
  File "/home/ls2017/.local/lib/python2.7/site-packages/falcon_unzip-0.1.0-py2.7.egg/EGG-INFO/scripts/fc_unzip.py", line 1, in <module>
    from falcon_unzip.unzip import main
  File "/home/ls2017/.local/lib/python2.7/site-packages/falcon_unzip-0.1.0-py2.7.egg/falcon_unzip/unzip.py", line 1, in <module>
    from falcon_kit import run_support as support
ImportError: No module named falcon_kit
Traceback (most recent call last):
  File "/usr/local/bin/fc_quiver.py", line 4, in <module>
    __import__('pkg_resources').run_script('falcon-unzip==0.1.0', 'fc_quiver.py')
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 746, in run_script
  File "build/bdist.linux-x86_64/egg/pkg_resources/__init__.py", line 1501, in run_script
  File "/home/ls2017/.local/lib/python2.7/site-packages/falcon_unzip-0.1.0-py2.7.egg/EGG-INFO/scripts/fc_quiver.py", line 1, in <module>
    from falcon_unzip.run_quiver import main
  File "/home/ls2017/.local/lib/python2.7/site-packages/falcon_unzip-0.1.0-py2.7.egg/falcon_unzip/run_quiver.py", line 1, in <module>
    from falcon_kit import run_support as support
```

The steps I did are as following:

1. I installed falcon_unzip on my local Ubuntu machine. Then I copied all output files of running falcon with my data to a directory.

2. edit input_bam.fofn as following:
```
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.3.subreads.bam
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.2.subreads.bam
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.1.subreads.bam
```
3. edit input.fofn as following:
```
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.3.subreads.fasta
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.2.subreads.fasta
/home/ls2017/files/testing/unzip_testdata/ecoli.frE01/data/m141013_011508_sherri_c100709962550000001823135904221533_s1_p0.1.subreads.fasta
```
4. edit fc_unzip.cfg as following:
```ini
[General]
job_type = local

[Unzip]
input_fofn= input.fofn
input_bam_fofn= input_bam.fofn

smrt_bin=/home/ls2017/files/testing/smrt/install/install/smrtlink-fromsrc_3.1.0.180439,180439-180439-180128-180128-180128/bundles/smrttools/smrtcmds/bin

jobqueue =
sge_phasing=
sge_quiver=
sge_track_reads=
sge_blasr_aln=
sge_hasm=
unzip_concurrent_jobs = 1
quiver_concurrent_jobs = 1
```

5. enter the directory in command line, run `./unzip.sh`, I got the above error.

**Comment by pb-cdunn** [comment]
> ImportError: No module named falcon_kit

You have to install FALCON in addition to FALCON_unzip.

**Comment by yingzhang121** [comment]
I interpreted the "ImportError" as the module is not in your PYTHONPATH.  Actually, I had the issue too, so I manually added the Falcon_unzip/src/build/lib etc to my pythonpath, and the issue got resolved.  Also I doubt whether other python 2.7 will work.  Because all the python scripts for Falcon Unzip have no shebang, a python environment is needed before any of the script could be run, and Falcon set up the environment.
